instructure/canvas-android
# [MBL-15473][Teacher][Student] Grab Annotation tool

Test plan:
- Verify that the user can select and move annotations when the grab tool is selected.
- Verify that if the user deselects the previously selected annotation tool the grab tool will be selected automatically.
- Smoke test all the annotation features in both apps.
- Test also MBL-14944, I removed that fix because it is not needed anymore.

refs: MBL-15473
affects: Teacher, Student
release note: Added grab annotation tool.
